README.md

```diff
@@ -50,26 +50,33 @@ in the napari plugin [napari-segment-blobs-and-things-with-membranes](https://ww
 
 ### Measurements
 The first step is deriving measurements from the labelled image and the corresponding pixels in the grey-value image.
-You can use the menu `Tools > Measurement > Measure intensity, shape and neighbor counts (ncp)` for that.
+Since the 0.6.0 release measurement is no longer part of this plugin and you will have to use other napari plugins to measure your data.
```

Comment:
```suggestion
Since the 0.6.0 release measurements widget is no longer part of this plugin and you will have to use other napari plugins to measure your data.
```